staging/cse/windows/provisioningscripts/cleanupnetwork.ps1

```diff
@@ -1,37 +1,42 @@
 $Global:ClusterConfiguration = ConvertFrom-Json ((Get-Content "c:\k\kubeclusterconfig.json" -ErrorAction Stop) | out-string)
+$CleanupHnsNetwork = [System.Convert]::ToBoolean($Global:ClusterConfiguration.Services.CleanupHnsNetwork) # CleanupHnsNetwork is legacy code
 
 $global:NetworkMode = "L2Bridge"
 $global:NetworkPlugin = $Global:ClusterConfiguration.Cni.Name
 $global:HNSModule = "c:\k\hns.v2.psm1"
 
 ipmo $global:HNSModule
 
-$networkname = $global:NetworkMode.ToLower()
-if ($global:NetworkPlugin -eq "azure") {
-    $networkname = "azure"
-}
+if ($CleanupHnsNetwork) { # CleanupHnsNetwork is legacy code
```

Comment:
Can we test this change in below scenarios?
1. kill hns service
2. kill containerd
3. restart kubelet
4. restart the node
5. ...